Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/LineTool.kt

```diff
@@ -85,6 +82,10 @@ class LineTool(
 
     var undoRecentlyClicked = false
 
+    var firstMove = true
+
+    var fromMove = false
```

Comment:
i suggest renaming these to variables to make there functionality more clear, especially `fromMove` does not give a lot of insight. `firstMove` could be renamed aswell to make it obvious that it's used for connected lines :smiley: